WcaOnRails/config/locales/en.yml

```diff
@@ -956,11 +966,24 @@ en:
         one: "WCA Delegate"
         other: "WCA Delegates"
       contact: "Contact"
-      competitor_limit: "Competitor Limit"
-      entry_fee: "Entry Fee"
       information: "Information"
       events: "Events"
       registration_requirements: "Registration requirements"
+      create_wca_account_html: "Create a WCA account %{here} if you don't have one."
+      claim_wca_id_html: "If this is not your first competition, associate your WCA ID to your WCA account %{here}."
+      register_html: "Register for this competition %{here}."
+      competitor_limit: "There is a competitor limit of %{competitor_limit} competitors."
```

Comment:
Re-using the same key name as above creates a very bad textual "bug": if the translation is not up to date the old text is displayed, leading to a very weird text where you have the field name without any number :s
I'd suggest either:
  - to remove this key in all translations (kind of long and boring...)
  - slightly change the key name, eg to `competitor_limit_is`